pkgs/development/compilers/pforth/default.nix

```diff
@@ -14,14 +15,25 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-vEjFeHSJl+yAtatYJEnu+r9hmOr/kZOgIbSUXR/c8WU=";
   };
 
-  dontConfigure = true;
```

Comment:
Why?

Reply:
Why not? this line does not do anything besides omitting a message and if the project eventually starts using a configuration script, we pick it up automatically. But I am not too hung up on this, and I can put it back if you want to.